src/somef/extract_software_type.py

```diff
@@ -0,0 +1,309 @@
+import os
+from pathlib import Path
+import nbformat
+from chardet import detect
+import re
+from .extract_workflows import is_file_workflow
+from .process_results import Result
+
+media_extensions = ('.rData','.bib','.csv','.xlx','.stan','.tex','.html','.css','.rst','.md','.png', '.jpg', '.jpeg', '.gif', '.bmp', '.svg', '.mp4', '.avi', '.mkv', '.wmv', '.flv', '.mov', '.mp3', '.wav', '.wma', '.flac', '.aac', '.ogg', '.pdf' )
+
+def check_repository_type(path_repo,title,metadata_result:Result):
+    """ Function that adds the metadata result in the JSON 
+        output depending on the software type"""
+    if check_extras(path_repo):
```

Comment:
this does not seem right. If it complies with check extras, then it is NOT software, but it is actually classified as software.

Reply:
based on our last discussion we concluded that the format should contain just the link to the definition  and type URL
"type": [{"result": {"value": "https://w3id.org/software-types#SoftwareLibrary", "type": "Url"}, "confidence": test results confidence, "technique": software type heuristics"}]
That function itself works with adding the metadata based on the type of repository and if its not software it will put the results with a link to what software actually is. I was thinking that in addition to what we talked to add in a metadata also the category ( inside the result another variable mentioning static website/extras/commandline etc.)